src/analysis/inner_product_space/pi_L2.lean

```diff
@@ -470,4 +473,18 @@ lemma inner_matrix_row_row (A B : matrix (fin n) (fin m) 𝕜) (i j : (fin n)) :
 lemma inner_matrix_col_col (A B : matrix (fin n) (fin m) 𝕜) (i j : (fin m)) :
   ⟪Aᵀ i, Bᵀ j⟫ₙ = (Aᴴ ⬝ B) i j := rfl
 
+variables (i : ι) (j : ι')
+
+lemma euclidean_space.mul_vec_single_apply [decidable_eq ι] (A : matrix ι' ι 𝕜) :
+  A.mul_vec (euclidean_space.single i 1) j = A j i :=
+matrix.mul_vec_std_basis A j i
```

Comment:
This lemma is a bit suspicious, as it's not type-correct; it ought to have a `pi_Lp.equiv` in there somewhere.